# Estonian translation for linuxmint
# Copyright (c) 2009 Rosetta Contributors and Canonical Ltd 2009
# This file is distributed under the same license as the linuxmint package.
# FIRST AUTHOR <EMAIL@ADDRESS>, 2009.
#
msgid ""
msgstr ""
"Project-Id-Version: linuxmint\n"
"Report-Msgid-Bugs-To: FULL NAME <EMAIL@ADDRESS>\n"
"POT-Creation-Date: 2016-04-22 13:59+0100\n"
"PO-Revision-Date: 2015-03-14 22:34+0000\n"
"Last-Translator: mahfiaz <mahfiaz@gmail.com>\n"
"Language-Team: Estonian <et@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"X-Launchpad-Export-Date: 2016-06-25 09:26+0000\n"
"X-Generator: Launchpad (build 18111)\n"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:111
#: usr/lib/linuxmint/mintbackup/mintbackup.py:252
#: usr/lib/linuxmint/mintbackup/mintbackup.py:449
#: usr/lib/linuxmint/mintbackup/mintbackup.py:467
#: usr/lib/linuxmint/mintbackup/mintbackup.py:533
#: usr/lib/linuxmint/mintbackup/mintbackup.py:536
#: usr/lib/linuxmint/mintbackup/mintbackup.py:588
#: usr/lib/linuxmint/mintbackup/mintbackup.py:591
#: usr/lib/linuxmint/mintbackup/mintbackup.py:611
#: usr/lib/linuxmint/mintbackup/mintbackup.py:634
#: usr/lib/linuxmint/mintbackup/mintbackup.py:641
#: usr/lib/linuxmint/mintbackup/mintbackup.py:1498
#: usr/lib/linuxmint/mintbackup/mintbackup.py:1596
#: usr/lib/linuxmint/mintbackup/mintbackup.py:1604
#: usr/lib/linuxmint/mintbackup/mintbackup.py:1656 generate_desktop_files.py:26
#: generate_desktop_files.py:41 generate_desktop_files.py:56
msgid "Backup Tool"
msgstr "Varundustööriist"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:169
msgid "Preserve structure"
msgstr "Ülesehitus säilitatakse"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:171
msgid ".tar file"
msgstr ".tar fail"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:172
msgid ".tar.bz2 file"
msgstr ".tar.bz2 fail"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:173
msgid ".tar.gz file"
msgstr ".tar.gz fail"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:179
msgid "Never"
msgstr "Mitte kunagi"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:180
msgid "Size mismatch"
msgstr "Suuruste sobimatus"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:181
msgid "Modification time mismatch"
msgstr "Muutuste aja sobimatus"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:182
msgid "Checksum mismatch"
msgstr "Kontrollsumma sobimatus"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:183
msgid "Always"
msgstr "Alati"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:205
msgid "Excluded paths"
msgstr "Välistatud asukohad"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:215
msgid "Type"
msgstr "Tüüp"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:219
msgid "Detail"
msgstr "Detail"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:225
#: usr/lib/linuxmint/mintbackup/mintbackup.py:234
msgid "Path"
msgstr "Asukoht"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:228
#: usr/lib/linuxmint/mintbackup/mintbackup.py:237
msgid "Error"
msgstr "Tõrge"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:269
msgid "Store?"
msgstr "Kas varundada?"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:272
#: usr/lib/linuxmint/mintbackup/mintbackup.py:285
msgid "Name"
msgstr "Nimi"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:282
msgid "Install"
msgstr "Paigalda"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:290
msgid "Backup or Restore"
msgstr "Varunda või taasta"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:291
msgid "Choose from the following options"
msgstr "Vali järgmiste suvandite seast"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:293
#: usr/lib/linuxmint/mintbackup/mintbackup.py:309
#: usr/lib/linuxmint/mintbackup/mintbackup.py:323
#: usr/lib/linuxmint/mintbackup/mintbackup.py:330
#: usr/lib/linuxmint/mintbackup/mintbackup.py:334
#: usr/lib/linuxmint/mintbackup/mintbackup.py:339
msgid "Backup files"
msgstr "Varunda failid"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:294
#: usr/lib/linuxmint/mintbackup/mintbackup.py:343
#: usr/lib/linuxmint/mintbackup/mintbackup.py:353
#: usr/lib/linuxmint/mintbackup/mintbackup.py:359
#: usr/lib/linuxmint/mintbackup/mintbackup.py:364
msgid "Restore files"
msgstr "Taasta failid"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:295
#: usr/lib/linuxmint/mintbackup/mintbackup.py:368
#: usr/lib/linuxmint/mintbackup/mintbackup.py:373
#: usr/lib/linuxmint/mintbackup/mintbackup.py:379
#: usr/lib/linuxmint/mintbackup/mintbackup.py:384
msgid "Backup software selection"
msgstr "Varunda tarkvaravalik"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:296
#: usr/lib/linuxmint/mintbackup/mintbackup.py:388
#: usr/lib/linuxmint/mintbackup/mintbackup.py:393
#: usr/lib/linuxmint/mintbackup/mintbackup.py:400
msgid "Restore software selection"
msgstr "Taasta tarkvaravalik"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:298
msgid "Make a backup of your files"
msgstr "Failidest varukoopia tegemine"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:299
msgid "Save the list of installed applications"
msgstr "Paigaldatud rakenduste nimekirja salvestamine"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:300
msgid "Restore a previous backup"
msgstr "Eelmise varukoopia taastamine"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:301
msgid "Restore previously installed applications"
msgstr "Varem paigaldatud rakenduste taastamine"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:310
msgid "Please select a source and a destination for your backup"
msgstr "Palun vali allikas ja sihtkoht oma varukoopiale"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:311
#: usr/lib/linuxmint/mintbackup/mintbackup.py:347
#: usr/lib/linuxmint/mintbackup/mintbackup.py:355
msgid "Source:"
msgstr "Lähtekoht:"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:312
#: usr/lib/linuxmint/mintbackup/mintbackup.py:348
msgid "Destination:"
msgstr "Sihtkoht:"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:313
#: usr/lib/linuxmint/mintbackup/mintbackup.py:349
msgid "Advanced options"
msgstr "Täpsemad valikud"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:314
#: usr/lib/linuxmint/mintbackup/mintbackup.py:356
msgid "Description:"
msgstr "Kirjeldus:"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:315
msgid "Output:"
msgstr "Väljund:"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:316
#: usr/lib/linuxmint/mintbackup/mintbackup.py:350
msgid "Overwrite:"
msgstr "Ülekirjutus:"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:317
msgid "Confirm integrity"
msgstr "Terviklikkuse kinnitamine"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:318
msgid "Follow symlinks"
msgstr "Sümbolviitade järgimine"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:319
msgid "Preserve permissions"
msgstr "Õiguste säilitamine"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:320
msgid "Preserve timestamps"
msgstr "Ajatemplite säilitamine"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:324
msgid "Please select any files or directories you want to exclude"
msgstr "Palun vali failid või kataloogid, mille soovid välja jätta"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:325
msgid "Exclude files"
msgstr "Välista failid"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:326
msgid "Exclude directories"
msgstr "Välista kataloogid"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:327
msgid "Remove"
msgstr "Eemalda"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:331
msgid "Please review the information below before starting the backup"
msgstr "Enne varundamise alustamist vaata järgnevad andmed üle"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:335
msgid "Please wait while your files are being backed up"
msgstr "Palun oota, kuni failid on varundatud"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:336
#: usr/lib/linuxmint/mintbackup/mintbackup.py:381
msgid "Backing up:"
msgstr "Varundamine:"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:340
#: usr/lib/linuxmint/mintbackup/mintbackup.py:385
msgid "The backup is now finished"
msgstr "Varundamine on valmis"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:344
msgid ""
"Please choose the type of backup to restore, its location and a destination"
msgstr "Palun vali taastatava varunduse tüüp, selle asukoht ja sihtkoht"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:345
msgid "Archive"
msgstr "Arhiiv"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:346
msgid "Directory"
msgstr "Kataloog"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:354
msgid "Please review the information below"
msgstr "Palun vaata järgnev informatsioon üle"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:360
msgid "Please wait while your files are being restored"
msgstr "Palun oota, kuni failid on taastatud"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:361
msgid "Restoring:"
msgstr "Taastamine:"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:365
msgid "The restoration of the files is now finished"
msgstr "Failide taastamine on valmis"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:369
msgid "Please choose a destination"
msgstr "Palun vali sihtkoht"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:370
#: usr/lib/linuxmint/mintbackup/mintbackup.py:552
msgid "Destination"
msgstr "Sihtkoht"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:374
msgid "The list below shows the packages you added to Linux Mint"
msgstr "Järgnev nimekiri näitab pakette, mille lisasid Linux Mintile"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:375
#: usr/lib/linuxmint/mintbackup/mintbackup.py:395
msgid "Select all"
msgstr "Vali kõik"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:376
#: usr/lib/linuxmint/mintbackup/mintbackup.py:396
msgid "Deselect all"
msgstr "Tühista kõik valikud"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:380
msgid "Please wait while your software selection is being backed up"
msgstr "Palun oota, kuni tarkvaravalik on varundatud"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:389
msgid "Please select a saved software selection"
msgstr "Palun vali salvestatud tarkvaravalik"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:390
msgid "Software selection:"
msgstr "Tarkvaravalik:"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:394
msgid "Select the packages you want to install"
msgstr "Vali paketid, mida soovid paigaldada"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:397
msgid "Refresh"
msgstr "Värskenda"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:401
msgid "The restoration is now finished"
msgstr "Taastamine on valmis"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:402
msgid "Your package selection was restored successfully"
msgstr ""

#: usr/lib/linuxmint/mintbackup/mintbackup.py:458
#: usr/lib/linuxmint/mintbackup/mintbackup.py:476
msgid "Invalid path"
msgstr "Vigane asukoht"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:458
#: usr/lib/linuxmint/mintbackup/mintbackup.py:476
#, python-format
msgid "%s is not located within your source directory. Not added."
msgstr "%s ei asu lähtekataloogis, seda ei lisatud."

#: usr/lib/linuxmint/mintbackup/mintbackup.py:533
msgid "Please choose directories for the source and the destination"
msgstr "Palun vali väljund ja sisendkausta asukoht"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:536
msgid ""
"Please choose different directories for the source and the destination"
msgstr "Palun vali erinevad kataloogid lähte- ja sihtkoha jaoks"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:551
msgid "Source"
msgstr "Lähtekoht"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:554
msgid "Description"
msgstr "Kirjeldus"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:558
msgid "Compression"
msgstr "Tihendamine"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:562
msgid "Overwrite destination files"
msgstr "Sihtfailide asendamine"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:565
msgid "Exclude"
msgstr "Välja jäetakse"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:588
msgid "Please choose a file to restore from"
msgstr "Palun vali fail, millest taastada"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:591
#: usr/lib/linuxmint/mintbackup/mintbackup.py:611
msgid "Please choose a destination directory"
msgstr "Palun vali sihtkataloog"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:634
msgid "No packages need to be installed at this time"
msgstr "Paigaldamist vajavad paketid hetkel puuduvad"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:641
msgid "Please select one or more packages to install"
msgstr "Vali paigaldamiseks üks või enam paketti"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:709
#: usr/lib/linuxmint/mintbackup/mintbackup.py:710
#: usr/lib/linuxmint/mintbackup/mintbackup.py:1204
#: usr/lib/linuxmint/mintbackup/mintbackup.py:1206
#: usr/lib/linuxmint/mintbackup/mintbackup.py:1513
#: usr/lib/linuxmint/mintbackup/mintbackup.py:1514
msgid "Calculating..."
msgstr "Arvutamine..."

#: usr/lib/linuxmint/mintbackup/mintbackup.py:759
#: usr/lib/linuxmint/mintbackup/mintbackup.py:799
msgid "Skipping broken link"
msgstr "Katkise lingi vahelejätmine"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:760
msgid "Broken link"
msgstr "Katkine link"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:763
#: usr/lib/linuxmint/mintbackup/mintbackup.py:803
msgid "Skipping link"
msgstr "Lingi vahelejätmine"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:827
#: usr/lib/linuxmint/mintbackup/mintbackup.py:836
#: usr/lib/linuxmint/mintbackup/mintbackup.py:845
#: usr/lib/linuxmint/mintbackup/mintbackup.py:1265
#: usr/lib/linuxmint/mintbackup/mintbackup.py:1276
#: usr/lib/linuxmint/mintbackup/mintbackup.py:1289
#: usr/lib/linuxmint/mintbackup/mintbackup.py:1354
#: usr/lib/linuxmint/mintbackup/mintbackup.py:1363
#: usr/lib/linuxmint/mintbackup/mintbackup.py:1373
msgid "Skipping identical file"
msgstr "Identse faili vahelejätmine"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:867
#, python-format
msgid ""
"Warning: Some files were not saved, copied: %(current_file)d files out of "
"%(total)d total"
msgstr ""
"Hoiatus: Mõnda faili ei salvestatud, kopeeritud %(current_file)d faili kokku "
"%(total)d-st"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:871
msgid "An error occurred during the backup"
msgstr ""

#: usr/lib/linuxmint/mintbackup/mintbackup.py:881
#: usr/lib/linuxmint/mintbackup/mintbackup.py:1559
msgid "The backup was aborted"
msgstr "Varundamine katkestati"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:889
msgid "The backup completed successfully"
msgstr "Varundamine lõpetati edukalt"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:980
msgid "Checksum Mismatch:"
msgstr "Kontrollsumma ei klapi:"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:981
msgid "Checksum Mismatch"
msgstr "Kontrollsumma ei klapi"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:1033
#: usr/lib/linuxmint/mintbackup/mintbackup.py:1035
#: usr/lib/linuxmint/mintbackup/mintbackup.py:1064
msgid "Calculating checksum"
msgstr "Kontrollsumma arvutamine"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:1110
#: usr/lib/linuxmint/mintbackup/mintbackup.py:1134
msgid "(Not created with the Backup Tool)"
msgstr "(ei ole loodud varunduse töörista poolt)"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:1397
#, python-format
msgid ""
"Warning: Some files were not restored, copied: %(current_file)d files out of "
"%(total)d total"
msgstr ""
"Hoiatus: Mõnda faili ei taastatud, kopeeritud: %(current_file)d faili "
"%(total)d-st"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:1400
msgid "An error occurred during the restoration"
msgstr ""

#: usr/lib/linuxmint/mintbackup/mintbackup.py:1411
msgid "The restoration was aborted"
msgstr "Taastamine katkestati"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:1419
msgid "The restoration completed successfully"
msgstr "Taastamine lõpetati edukalt"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:1550
msgid "An error occurred during the backup:"
msgstr "Tekkis tõrge varunduses:"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:1567
msgid "Your software selection was backed up successfully"
msgstr ""

#: usr/lib/linuxmint/mintbackup/mintbackup.py:1596
msgid "The specified file is not a valid software selection"
msgstr "Määratud fail pole korrektne tarkvaravalik"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:1604
#: usr/lib/linuxmint/mintbackup/mintbackup.py:1656
msgid "An error occurred while accessing the file"
msgstr "Failile ligipääsu hankimise käigus tekkis viga"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:1649
msgid "Could not locate the package"
msgstr "Paketti ei suudetud leida"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:1681
msgid "Please wait, this can take some time"
msgstr "Palun oota, selleks kulub natuke aega"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:1683
msgid "The installation is complete"
msgstr "Paigaldamine on lõpetatud"

#: generate_desktop_files.py:26 generate_desktop_files.py:41
#: generate_desktop_files.py:56
msgid "Make a backup of your home directory"
msgstr "Kodukataloogist varukoopia tegemine"

#~ msgid "Cleaning up..."
#~ msgstr "Puhastamine..."

#~ msgid "Your backup was successfully restored"
#~ msgstr "Teie varundus on edukalt taastatud"

#~ msgid "Backup"
#~ msgstr "Varundamine"

#~ msgid "Restore"
#~ msgstr "Taasta"

#~ msgid "View content"
#~ msgstr "Vaata sisu"

#~ msgid "is not located within your home directory. Not added."
#~ msgstr "ei asu teie kodukataloogis. Ei lisata."

#~ msgid "Included"
#~ msgstr "Kaasa arvatud"

#~ msgid "Excluded Files and Directories"
#~ msgstr "Välistatud failid ja kataloogid"

#~ msgid "Exclude folders"
#~ msgstr "Välistatud kaustad"

#~ msgid "Restoration failed"
#~ msgstr "Taastamine ebaõnnestus"

#~ msgid "Extracting the content of the backup into your home directory..."
#~ msgstr "Varukoopia sisu lahtipakkimine kodukataloogi..."

#~ msgid "Archiving your home directory..."
#~ msgstr "Kodukataloogi arhiveerimine...."

#~ msgid "Copying the backup file into your home directory..."
#~ msgstr "Varukoopia failide kopeerimine kodukataloogi..."

#~ msgid "An error occurred while restoring the backup archive:"
#~ msgstr "Varukoopia arhiivi taastamisel esines viga:"

#~ msgid "Opening the backup archive..."
#~ msgstr "Varundusarhiivi avamine..."

#~ msgid "Restore your personal data from this backup"
#~ msgstr "Taasta oma isiklikud andmed sellest varundusest"

#~ msgid "<big><b>Load data into your home directory</b></big>"
#~ msgstr "<big><b>Laadi andmed oma kataloogi</b></big>"

#~ msgid "Included hidden directories"
#~ msgstr "Kaasa arvatud varjatud kataloogid"

#~ msgid "Overwrite existing files"
#~ msgstr "Kirjuta olemasolev fail üle"

#~ msgid "Your home directory was successfully backed-up into"
#~ msgstr "Kodukataloogi sihtkoht varundati edukalt"

#~ msgid "Backup successful"
#~ msgstr "Varundus õnnestus"

#~ msgid "Backup failed"
#~ msgstr "Varundus ebaõnnestus"

#~ msgid "Restoration successful"
#~ msgstr "Taastamine õnnestus"

#~ msgid "Read error"
#~ msgstr "Viga lugemisel"

#~ msgid "An error occurred while opening the backup:"
#~ msgstr "Varunduse avamisel tekkis viga:"

#~ msgid "Hidden paths"
#~ msgstr "Varjatud asukohad"
